Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Stage 1: Build
+FROM node:16 AS build
+
+WORKDIR /code
+
+# Copy package.json and package-lock.json (if available)
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the rest of the source code
+COPY . .
```

Comment:
Note that Copy statement will copy all files in folder no include in .dockerignore